Makefile

```diff
@@ -122,22 +122,16 @@ diff_cover: test
 
 ## Localization targets
 
-extract_translations: ## extract strings to be translated, outputting .mo files
+extract_translations: ## extract strings to be translated, outputting .po files
 	cd edx_proctoring && django-admin makemessages -l en -v1 -d django
 	cd edx_proctoring && django-admin makemessages -l en -v1 -d djangojs
 
-compile_translations: ## compile translation files, outputting .po files for each supported language
+compile_translations: ## compile translation files, outputting .mo files for each supported language
 	cd edx_proctoring && django-admin compilemessages
 
 detect_changed_source_translations:
 	cd edx_proctoring && i18n_tool changed
 
-pull_translations: ## pull translations from Transifex
```

Comment:
This appears to only be being used by the [push and pull Transifex ecommerce scripts](https://github.com/openedx/ecommerce-scripts/tree/master/transifex).